fpo-api/api/pdf.py

```diff
@@ -61,7 +61,7 @@ def transform_data_for_pdf(original_data):
         pass
 
     try:
-        data["hearingLocation"] = Location.objects.get(id=hearingLocationId).name
+        data["hearingLocationPDF"] = Location.objects.get(id=hearingLocationId).name
```

Comment:
Is the variable change from `hearingLocation` to `hearingLocationPDF` necessary?